frame/staking/reward-curve/src/lib.rs

```diff
@@ -268,14 +268,20 @@ impl INPoS {
 		}
 	}
 
+	// calculates x_ideal from:
+	// y = i_0 + (i_ideal * x_ideal - i_0) * 2^((x_ideal - x)/d)
+	// See web3 docs for the details
 	fn compute_opposite_after_x_ideal(&self, y: u32) -> u32 {
 		if y == self.i_0 {
 			return u32::max_value();
 		}
+		// Note: the log term calculated here represents a negative per_million value
```

Comment:
considering previous comment this term is not a negative value of `log2((y - self.i_0) / (self.i_ideal_times_x_ideal - self.i_0))` but a positive value of `log2((self.i_ideal_times_x_ideal - self.i_0) /( y - self.i_0));`
```suggestion
		// Note: the log term calculated here represents a per_million value
```